aten/src/ATen/native/mps/operations/Blas.mm

```diff
@@ -20,6 +20,9 @@ Tensor dot_mps(
   const Tensor &self,
   const Tensor &other)
 {
+
+  TORCH_CHECK(self.scalar_type() != ScalarType::Long, "MPS: dot op doesn't support int64 input")
```

Comment:
missing semicolon
```suggestion
  TORCH_CHECK(self.scalar_type() != ScalarType::Long, "MPS: dot op doesn't support int64 input");
```

Reply:
You don't need to have semicolon for this.